pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -170,6 +190,21 @@ func CreateOrLoadClusterInfo(clusterdContext *clusterd.Context, context context.
 	return clusterInfo, maxMonID, monMapping, nil
 }
 
+func genAdminKeyrings(context *clusterd.Context, namespace string) (*cephclient.CephCred, error) {
```

Comment:
Since we're not generating the toolbox keyring here:
```suggestion
func genAdminKeyring(context *clusterd.Context, namespace string) (*cephclient.CephCred, error) {
```